Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Political Factions Game Rule

### Changelog:
- Added the Political Factions game rule that regulates the size and existence of the political factions. Has 4 options:
  - Medium (The AI doesn't join factions that have reached the limit of 200 Realm Size and 10 Faction Tributaries)
  - Large (The AI doesn't join factions that have reached the limit of 300 Realm Size and 15 Faction Tributaries)
  - Off (cannot be created)
  - Small (The AI doesn't join factions that have reached the limit of 100 Realm Size and 5 Faction Tributaries)

**_P.S. The AI can't join on its own, but you still can invite it._**

### How to Test
- Try to create the Alliance as Terenas or the Horde as Blackhand in the first bookmark. Try to play Terenas or Thrall in the second bookmark. See if the rule works.
- Try to disable the factions and fire the Exodus of the Horde event by using `event WCTHW.120`, your character should have the Horde title, but not the Leader of the Horde trait.
- Try to disable the factions and look at the Alliance in the second bookmark. It should be destroyed after Arthas surrenders to the Lich King.
